presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/HiveCommitHandle.java

```diff
@@ -30,21 +30,42 @@
 public class HiveCommitHandle
         implements ConnectorCommitHandle
 {
-    public static final HiveCommitHandle EMPTY_HIVE_COMMIT_HANDLE = new HiveCommitHandle(ImmutableMap.of());
+    public static final HiveCommitHandle EMPTY_HIVE_COMMIT_HANDLE = new HiveCommitHandle(ImmutableMap.of(), ImmutableMap.of());
 
     private final Map<SchemaTableName, List<DateTime>> lastDataCommitTimes;
+    private final Map<SchemaTableName, List<DateTime>> lastDataCommitTimesForWrite;
```

Comment:
We probably need specific comments to elaborate the differences between these two fields